### ISSUE TITLE
Reporting errors on /private/tmp/flow/.../bom.js

### ISSUE THREAD

**Issue by Emilios1995** [facebook/flow#4760]
I don't know what this file is or why is flow checking it. But it's reporting 3 errors there.

<img width="1080" alt="screen shot 2017-08-31 at 1 37 09 pm" src="https://user-images.githubusercontent.com/12430923/29941192-53d5f684-8e57-11e7-8c72-acec41ec7446.png">

flow version: 0.53.1
os: macos

**Comment by asolove** [comment]
Does your application use the geolocation browser api at all? Can you share anything else about the code, especially anything that calls getCurrentPosition? Looks like that’s what the error is about.

The error messages are definitely showing up in the wrong place, but probably refer to something that is actually a type error in your code. I’m happy to help debug them if I can get some more information. You can try running ‘flow check —traces 10’ to see more context on why flow thinks this is an error.

**Comment by Emilios1995** [comment]
Yes, I'm using react-native and its geolocation API. I'll look for type errors in the places where I use it and report back if I can't find any.
Thanks

**Comment by Emilios1995** [comment]
@asolove I already located and corrected the error. I was calling `getCurrentPosition` without those three properties in the options arguments (enableHighAccuracy, timeout, maximumAge).

Should this issue be closed? I mean, I would have liked to see the error in the correct file.

**Comment by asolove** [comment]
Yeah, I’d like to leave it open. This seems like a pretty easy case for the error message to be in the right place and yet it’s wrong.

**Comment by asolove-stripe** [comment]
Can you share the code that caused this? Or try again with Flow 0.55? There are big improvements in error locations in the new version.
